static/absolute/js/searchPage.js

```diff
@@ -147,10 +160,44 @@ function showRestaurants(restaurantsResponse) {
     return;
   }
 
+  //Used to get additional information about the restaurant results.
+  const fields = "url,formatted_phone_number,website,review";
+
   //This will be used to create a new div for every restaurant returned by the Places Library:
-  allRestaurants.results.forEach((restaurant) => {
-    let restaurantDiv = document.createElement("div");
-    restaurantDiv.classList.add("restaurant-card");
+  let restaurantIndex = 1;
+  for (restaurant of allRestaurants.results) {
+    //Get additional details for every restaurant.
+    let placeDetailsResponse = await (
+      await fetch(
+        `/api/placedetails?fields=${fields}&id=${restaurant.place_id}`
+      )
+    ).json();
+
+    let additionalDetails =
+      placeDetailsResponse.status === 200
+        ? placeDetailsResponse.data.result
+        : {};
+
+    //Create a restaurant card to hold information about each restaurant.
+    let infoDiv = document.createElement("div");
+    infoDiv.classList.add("info-div");
+
+    //Create a section to hold the image
+    try {
+      let imageDiv = document.createElement("div");
+      // if (placePhotosResponse.hasOwnProperty("data")) {
+      let image = document.createElement("img");
+      image.src =
+        "https://maps.googleapis.com/maps/api/place/photo?photoreference=" +
+        restaurant.photos[0].photo_reference +
```

Comment:
nit: making this in such a way that its easier to understand it's coming from the place search response. Maybe placing it in a descriptive variable name?

Reply:
Done.